deployment/secure_research_desktop/cloud_init/cloud-init-buildimage-ubuntu-2004.mustache.yaml

```diff
@@ -405,6 +405,12 @@ runcmd:
   # Run installation tests
   - python3 /opt/build/check_installation.py || die "Installation check failed!";
 
+  # Remove Linux omsagent before generalising the image
+  - echo ">=== $(date +%s) Removing Linux omsagent ===<"
+  - wget https://raw.githubusercontent.com/Microsoft/OMS-Agent-for-Linux/c7e1a8f923844f989c8aa7957cee0e89708d821f/installer/scripts/onboard_agent.sh
```

Comment:
```suggestion
  # We use an explicit tag (currently 1.17.2-0) so that we can audit what the script is doing
  - wget https://raw.githubusercontent.com/microsoft/OMS-Agent-for-Linux/OMSAgent_v1.17.2-0/installer/scripts/onboard_agent.sh
```

Reply:
Thoughts @craddm @JimMadge ?